src/haddock/gear/read_io.py

```diff
@@ -0,0 +1,22 @@
+"""Gear to handle the reading of the IO."""
```

Comment:
If this file is so heavily dependent on libs.libontology and is something we forsee a core part of the workflow, it should go to `libontology` directly.

Reply:
Your call